### PR TITLE
Update CellectController responses to be more usable

### PR DESCRIPTION
**Pull request by parrish**
Instead of just returning ids, it returns the necessary fields for cellect.

### CODE REVIEW

**Comment by parrish** [comment]
There's something odd going on in the specs with set_member_subjects vs subjects.  Any thoughts?

File: app/controllers/cellect_controller.rb [hunk 1]
```diff
@@ -25,33 +25,35 @@ def launched_workflows
...
     end
   end
 
-  def cellect_workflow_subject_ids
+  def cellect_workflow_subjects
     if workflow = cellect_workflow_from_param
-      SetMemberSubject.non_retired_for_workflow(workflow).pluck(:subject_id)
+      SetMemberSubject
+        .non_retired_for_workflow(workflow)
+        .select(:id, :priority)
```
Comment (camallen) [inline app/controllers/cellect_controller.rb]: This should be `.select(:subject_id, :priority)` instead of the sms ids.

**Comment by camallen** [comment]
This is great - thanks @parrish. Yeah re the specs, it'll be the sms.id attribute select vs the sms.subject_id.

File: app/controllers/cellect_controller.rb (hunk 1 above)
Comment (parrish) [inline app/controllers/cellect_controller.rb]: Yep, that got it.  Thanks!

**Pull request merged**